docs/articles_en/openvino_workflow/gen_ai.md

```diff
@@ -0,0 +1,144 @@
+# Optimize and Deploy Generative AI Models {#gen_ai_guide}
+
+@sphinxdirective
+
+OpenVINO toolkit could be used to accelerate Generative AI use cases as those mostly rely on model inference. When it comes to Generative Models, OpenVINO supports:
+
+* Conversion, optimization and inference for Text, Image and Audio generative models (for example, llama2, MPT, OPT, Stable Diffusion, Stable Diffusion XL, etc.) 
+* Int8 weights compression for text generation models 
+* Reduced storage format (fp16 precision for non-compressed models and int8 for compressed models) 
+* Inference on CPU and GPU platforms, including Intel integrated and discrete ARC GPUs 
+ 
+
+There are two main paths how OpenVINO could be used to run Generative AI use cases:
+
+* As a backend for Hugging Face frameworks (transformers, diffusers) with use of `Optimum-Intel <https://huggingface.co/docs/optimum/intel/inference>`__ extension 
+* With use of OpenVINO native APIs (Python and C++) with use of custom pipeline code 
+
+ 
+In both cases, OpenVINO runtime and tools will be used, difference is mostly in API that you prefer and footprint of final solution. Use of native APIs allows using generative models in C++ applications, having minimal runtime dependencies and minimizing application footprint. Approach with OpenVINO Native APIs will require implementation of glue code (generation loop, text tokenization or scheduler functions) which is hidden as implementation detail internally within Hugging Face libraries. 
+
+We recommend starting with use of Hugging Face frameworks, initially experimenting with different models and scenarios, ding your fit and then convert to OpenVINO native APIs if there is a demand.  
+
+Within Optimum-Intel product there are interfaces that allow to optimize model with use of `Neural Network Compression Framework (NNCF) <https://github.com/openvinotoolkit/nncf>`__ (compress weights) and export model to OpenVINO model format to be used in native API applications. 
+
+Table below summarizes differences of both Hugging Face and Native approaches. 
+
+.. list-table::
+   :widths: 20 25 55
+   :header-rows: 1
+
+   * -  
+     - Hugging Face through OpenVINO
+     - OpenVINO Native API
+   * - Model support
+     - Wide set of Models
+     - Wide set of Models
+   * - APIs
+     - Python (Hugging Face API)
+     - Python, C++ (OpenVINO API)
+   * - Model Format
+     - Source Framework
+     - OpenVINO
+   * - Inference code
+     - Hugging Face based
+     - Custom inference pipelines
+   * - Time to inference
+     - Slow
+     - Fast
+   * - Additional dependencies
+     - As much as Hugging Face has
+     - Numpy
```

Comment:
```suggestion
   * - Additional Python dependencies
```
openvino-telemetry is also a dependency of OpenVINO runtime Python API. And for C++, we require some libraries to be installed too. These libraries are included in the Python wheels, but when not using wheels they need to be installed.

Reply:
In fact, telemetry is a part of OpenVINO distribution that is just extracted into a separate package for convenience. I agree that it does not sound fully correct. Maybe we should rephrase it to "lightweight (e.g. numpy, boost,, etc.)". @yury-gorbachev, what do you think?

Reply:
I agree- it should not be mentioned, this is part of OV distribution.